### DISCUSSION TITLE
I don't understand how to record actions

### DISCUSSION THREAD

**Discussion by tellts** [AutomaApp/automa#1621] [General]
There is a lesson on YouTube. People there asked a question many months ago about the same error as mine. I just wrote there too. The lesson is called
Autorial: Part 1. Record Workflow - Level Easy
When I start work it’s different for me. When I click, the recording does not start, but I see the following text and I am redirected to another page in docs.
--------------------------------
Start recording by opening the dashboard. Click to learn more

**Comment by Delusionnnn** [comment]
Just get used to not using it, you'll need to at some point. Manually creating it is much better and precise. I could never get the record feature to work either because it would miss alot of my actions I needed.
